src/server/create.ts

```diff
@@ -24,7 +24,7 @@
  ******/
 
 // workaround for lack of typescript types for mojaloop dependencies
-// eslint-disable-next-line @typescript-eslint/no-triple-slash-reference
+// eslint-disable-next-line @typescript-eslint/triple-slash-reference
```

Comment:
if ambient.d.ts file is in root directory we should be able to drop `<reference path`

Reply:
existing linting issues will be fixed in separate story.